src/models/wikimedia/wikipedia/templates/wikipedia_page_reference.py

```diff
@@ -720,7 +720,7 @@ def __get_numbered_persons__(
         # Mypy warns that the following could add None to the list,
         # but that cannot happen.
         return [
-            self.__get_numbered_person__(
+            self.__get_numbered_person__(  # type: ignore
```

Comment:
I think here we should undo the list comprehension and then get the Person once and then add them to the result list if they are not None.